### ISSUE TITLE
Inline SVGs should be improved for accessibility considerations...

### ISSUE THREAD

**Issue by rdwatters** [gohugoio/gohugoioTheme#66]
We should add `<title>` and `<desc>` at least. See this handy CSS tricks article:

https://css-tricks.com/accessible-svgs/

Ref #65
